SDL2: freetype test failures

```
..............F...F.........F..............
======================================================================
FAIL: test_freetype_Font_render_mono (test.freetype_test.FreeTypeFontTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\David\Documents\dev\pygame\test\freetype_test.py", line 744, in test_freetype_Font_render_mono
    self.assertEqual(surf.get_alpha(), translucent_color.a)
AssertionError: None != 55

======================================================================
FAIL: test_freetype_Font_render_to_mono (test.freetype_test.FreeTypeFontTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\David\Documents\dev\pygame\test\freetype_test.py", line 804, in test_freetype_Font_render_to_mono
    self.assertEqual(surf.get_at(bottomright), r_fg_color)
AssertionError: (30, 23, 50, 255) != (30, 22, 49, 253)

======================================================================
FAIL: test_issue_144 (test.freetype_test.FreeTypeFontTest)
Issue #144: unable to render text
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\David\Documents\dev\pygame\test\freetype_test.py", line 1273, in test_issue_144
    self.assertRaises(pygame.error, font.render, s, (0, 0, 0))
AssertionError: error not raised by render

----------------------------------------------------------------------
Ran 43 tests in 3.325s

FAILED (failures=3)
```

**Related Docs:** 

https://www.pygame.org/docs/ref/freetype.html
https://www.pygame.org/docs/ref/surface.html#pygame.Surface.blit

--- a/test/freetype_test.py
+++ b/test/freetype_test.py
@@ -880,9 +880,6 @@ class FreeTypeFontTest(unittest.TestCase):
         finally:
             font.antialiased = save_antialiased
 
-    @unittest.skipIf(
-        pygame.get_sdl_version()[0] == 2, "skipping due to blending issue (#864)"
-    )
     def test_freetype_Font_render_to_mono(self):
         # Blitting is done in two stages. First the target is alpha filled
         # with the background color, if any. Second, the foreground

